Control/DeepSeq.hs

```diff
@@ -907,12 +902,6 @@ instance NFData m => NFData (WrappedMonoid m) where rnf = rnf1
 -- |@since 1.4.3.0
 instance NFData1 WrappedMonoid where
   liftRnf r (WrapMonoid a) = r a
-
--- |@since 1.4.2.0
-instance NFData a => NFData (Option a) where rnf = rnf1
--- |@since 1.4.3.0
-instance NFData1 Option where
-  liftRnf r (Option a) = liftRnf r a
```

Comment:
Ah, wait. We probably only want to remove these instances for GHC >= 9.2, but keep them for older GHCs!